integration/images.go

```diff
@@ -182,6 +182,11 @@ func (d *DockerFileBuilder) BuildImage(imageRepo, gcsBucket, dockerfilesPath, do
 		}
 	}
 
+	benchmarkFile := ""
```

Comment:
What do you think of enabling this as an environment variable rather than a flag?

Reply:
just for the integration tests? or in general?

Reply:
In general.